packages/extension/src/store/plugins/sync.js

```diff
@@ -106,6 +108,10 @@ const plugin = (store) => {
       // TODO: handle error
       await syncSettings(state);
       break;
+    case 'ui/setOpenNewTab':
```

Comment:
You can add this case to the above block of settings.
It's better to concentrate all the settings-based mutations in one place.